tests/ws_integration.rs

```diff
@@ -340,62 +341,58 @@ async fn live_live_endpoint() -> Result<(), Box<dyn std::error::Error>> {
 		}))
 		.unwrap(),
 	)
-	.await?;
+	.await;
+
+	assert!(live_res.is_ok(), "Unexpected error from the LIVE method: {:?}", live_res);
```

Comment:
`live_res.unwrap` already does this, no? Happy to keep it, but I've been caught by the is ok before where I forgot to add a message and the unwrap essentially gives the message as well

Reply:
yeah, you are right. I adding the asserts where the result is an Option and added this one by mistake too. Let me change it!